Sources/Networking/Core/APIManager.swift

```diff
@@ -7,7 +7,37 @@
 
 import Foundation
 
-/// Default API manager
+/** Default API manager which is responsible for the creation and management of network requests.
```

Comment:
Why are we using the `/**` here instead of staying with `///`?

Reply:
I've used it only for longer descriptions, especially with code examples etc. It's just easier to edit and keep the proper spacing that way.

Reply:
Got it, I would still prefer the `///` everywhere, but we can go with this 👍